### PR TITLE
Added Enemy Feedback

### PR DESCRIPTION
**Pull request by desultisoft**
I added an enemy flash effect and knockback when the enemy gets hit.

### CODE REVIEW

File: src/Code/Enemies/Hitbox.gd [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+	if invincible == true:
+		emit_signal("invincibility_started")
+	else:
+		emit_signal("invincibility_ended")
+
+func start_invincibility(duration):
+	self.invincible = true
+	timer.start(duration)
+
+func _on_Timer_timeout():
```
Comment (OceanBoy612) [inline src/Code/Enemies/Hitbox.gd:26]: I don't understand why timer and hitbox are cap

File: src/Code/Attacks/Attack/Attack.gd [hunk 2]
```diff
@@ -1,6 +1,6 @@
 extends Area2D
 
-
+export (float, 0,200,5) var knockback = 100;
```
Comment (OceanBoy612) [inline src/Code/Attacks/Attack/Attack.gd:3]: This is trash. Plz fix.

**Review by OceanBoy612 — CHANGES_REQUESTED** [review]
Most of this is looking good. Fix the broken trash and merge.

File: src/Code/Enemies/Hitbox.gd (hunk 1 above)
Comment (desultisoft) [reply src/Code/Enemies/Hitbox.gd:26]: These are Signal Functions, and if you see this format, assume so in the future as these aren't worth the time.

File: src/Code/Attacks/Attack/Attack.gd (hunk 2 above)
Comment (desultisoft) [reply src/Code/Attacks/Attack/Attack.gd:3]: You're trash Alex.

File: src/Code/Enemies/Enemy.gd [hunk 3]
```diff
@@ -59,3 +64,13 @@ func _get_dir_to_target() -> Vector2:
 		if t.length() < 10: return Vector2()
 		return t.normalized()
 	return Vector2()
+
+
+func _on_Hitbox_invincibility_started():
+	print("StartInvincibility")
```
Comment (Flamebeard10339) [inline src/Code/Enemies/Enemy.gd:70]: Remove extra print statements. You can use push_error or push-warning if this is something that should be tracked.

File: src/Code/Enemies/Enemy.gd [hunk 4]
```diff
@@ -59,3 +64,13 @@ func _get_dir_to_target() -> Vector2:
...
+	animationPlayer.play("Start")
+
+
+func _on_Hitbox_invincibility_ended():
+	print("StopInvincibility")
```
Comment (Flamebeard10339) [inline src/Code/Enemies/Enemy.gd:75]: Remove extra print statements. You can use push_error or push-warning if this is something that should be tracked.

**Review by Flamebeard10339 — COMMENTED** [review]
Good job

**Pull request merged**